index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hola Microverse</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <h3>¡¡Hola Microverse!!</h3>
+</html>
+</body>
```

Comment:
- Kindly correct the closing tags order of the HTML elements to create better code. Follow the [HTML best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).